### PR TITLE
Increase limit message chars 

### PR DESCRIPTION
**Pull request by ghost**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.6.1.X
| Description?  | Limit 1600 chars  we don't write SMS or tweet :)
| Type?         | bug fix 
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Write a message :)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8429)

### CODE REVIEW

**Comment by ghost** [comment]
You smoke what preston ?

**Comment by mickaelandrieu** [comment]
> You smoke what preston ?

Please read carefully the pull request description next time :)

File: classes/Message.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ class MessageCore extends ObjectModel
         'table' => 'message',
         'primary' => 'id_message',
         'fields' => array(
-            'message' =>        array('type' => self::TYPE_STRING, 'validate' => 'isCleanHtml', 'required' => true, 'size' => 1600),
+            'message' =>        array('type' => self::TYPE_STRING, 'validate' => 'isCleanHtml', 'required' => true, 'size' => 6400),
```
Comment (mickaelandrieu) [inline classes/Message.php:59]: are you sure the field in database can handle 6400 characters?

Comment (okom3pom) [reply classes/Message.php]: Oups in my own shop yes 👍 i forgot 1.6.1.18.sql

I create a new one because i do this comit directly on github.com

Comment (ghost) [reply classes/Message.php:59]: It's a text type

**Comment by LittleBigDev** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[classes/Message.php, line 59 at r1](https://reviewable.io:443/reviews/prestashop/prestashop/8429#-Kwo_bgotEU08gq57UAW-r1-59:-Kz3F9cxDD6Y4_S355Bd:b-7pabyo) ([raw file](https://github.com/prestashop/prestashop/blob/4512a7b84c6da4f8775e62ff9e2766aae99301d5/classes/Message.php#L59)):*
<details><summary><i>Previously, okom3pom wrote…</i></summary><blockquote>

It's a text type
</blockquote></details>

Indeed. Text mysql fields can store up to 65 535 characters (less if characters are encoded on 2 bytes)

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/prestashop/prestashop/8429#-:-Kz3GQUCYRD0YB4XbJsR:bnfp4nl)*

**Review by LittleBigDev — DISMISSED** [review]
Thank you @okom3pom !
Technically ok. But needs approval from product team (maybe design issues, as Message class is used in multiple places)
Ping @vincentbz ?

**Comment by ghost** [comment]
I did not find another place.

And after reflexion 6400 characters is not enough.

I have hook for refund exceeds 6400 characters.

**Comment by ghost** [comment]
Merchant can also put a notice in this field

**Comment by ghost** [comment]
@LittleBigDev i change commit name by a good one, i lost your review

**Review by LittleBigDev — APPROVED** [review]

File: classes/Message.php (hunk 1 above)
Comment (Quetzacoalt91) [reply classes/Message.php:59]: It seems MySQL can handle 65536 chars (= 2^16)

https://dev.mysql.com/doc/refman/5.7/en/storage-requirements.html#data-types-storage-reqs-strings

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @okom3pom

**Pull request merged**